ls1intum/Artemis
# [Chore] Adding role access right overview to the documentation

### Checklist

### Motivation and Context


The recent introduction of the editor role added complexity to the role system of Artemis.
To keep an overview of the permissions and the purpose of each role a new section is added to the documentation.

### Description

The section `Access Rights` should help understand when to use which role and display which functionality can be accessed by which role.

### Steps for Testing

Once you have installed Sphinx (it does not take long) you can locally create the documentation by executing `make html` in the `docs` folder. 

1. Build the documentation on your machine
2. Check whether all access rights are displayed correctly
3. Check whether access rights are missing
4. Did the documentation help you to understand when to use which role? Are there any questions left open that should be addressed in this section?
5. Check for spelling mistakes
